.templates/new-module/tests/react/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	entry: './index.jsx',
+	output: {
+		filename: './bundle.js'
+	},
+	module: {
+		rules: [
+			{
+				test: /\.jsx$/,
```

Comment:
fair enough, been struggeling to use `jsx` myself. Maybe we just move all together to `.js`?